modules/imgproc/test/test_color.cpp

```diff
@@ -2190,3 +2190,14 @@ TEST(ImgProc_Bayer2RGBA, accuracy)
         }
     }
 }
+
+TEST(ImgProc_BGR2RGBA, regression)
+{
+    Mat src(Size(10, 10), CV_8UC4);
```

Comment:
1. Please use issue number in regression test: `regression_8696`
2. It is better to use bigger size to enable checks for vectorized code too: `Size(80,10)`